core/driver/sources/src/main/resources/reference.conf

```diff
@@ -52,18 +62,106 @@ probe {
 
   // `resolvers` is for `org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers` - default values below
   resolvers {
-    intellij {
-      repositories = []
-    }
-    // default configuration for `plugins` is Plugins(None) - adding possible parameters in comments for reference
-    plugins {
-      // `repository.uri` is the URI of the plugin repository - use this config if you want to use a non-standard
-      // plugin repository. The standard one used by ide-probe is: https://plugins.jetbrains.com/plugin/download
-      # repository.uri = ""
-    }
-    jbr {
-      repositories = []
-    }
+
+    // `artifact` contains the name of the JetBrains product that should be used as the testing instance.
+    // For now, only IntelliJ IDEA is supported. There are two possible values:
+    // 1) "ideaIC" - for the IntelliJ IDEA Community edition  (default option)
+    // 2) "ideaIU" - for the IntelliJ IDEA Ultimate Edition
+    artifact = "ideaIC"
```

Comment:
You could just keep the artifact in patterns as it was. It is beyond scope of this PR. I am not sure if artifact is best name in the config (it makes sense in resolver patterns though). Also scope here is too wide. There is data for IJ and data for plugin, also for Jbr, and. this is just some dangling global artifact. We would probably add a new grouping key or make an aware breaking change to generalize it further. I don't think it is worth doing now and I don't want it to end up with a bad design.

Reply:
Ok, removed `probe.resolvers.artifact` config from code. `[artifact]` replacement should work just like before